### PR TITLE
adding top bottom coding info and logic

### PR DESCRIPTION
**Pull request by SPTKL**

### CODE REVIEW

File: factfinder/calculate.py [hunk 1]
```diff
@@ -266,9 +268,17 @@ def calculate_c_e_m_p_z(self, pff_variable: str, geotype: str) -> pd.DataFrame:
                     df["p"] = df.apply(
                         lambda row: get_p(row["e"], row["agg_e"]), axis=1
                     )
+
+                    def _get_z(name, e, m, p, agg_e, agg_m):
+                        results = get_z(e, m, p, agg_e, agg_m)
+                        if name == '36085029105':
```
Comment (mgraber) [inline factfinder/calculate.py]: Should we remove this?

File: factfinder/data/acs/2010/median.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
     "mdage": {
         "design_factor": 1.1,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:4]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 3]
```diff
@@ -98,6 +100,8 @@
     },
     "mdhhinc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:103]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 4]
```diff
@@ -167,6 +171,8 @@
     },
     "mdfaminc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:174]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 5]
```diff
@@ -236,6 +242,8 @@
     },
     "mdnfinc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:245]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 6]
```diff
@@ -305,6 +313,8 @@
     },
     "mdewrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:316]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 7]
```diff
@@ -390,6 +400,8 @@
     },
     "mdemftwrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:403]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 8]
```diff
@@ -475,6 +487,8 @@
     },
     "mdefftwrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:490]: Should be true

File: factfinder/data/acs/2010/median.json [hunk 9]
```diff
@@ -601,6 +617,8 @@
     },
     "mdvl": {
         "design_factor": 1.4,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:620]: I believe this should be true, but it looks like the top coding threshold is not the lower boundary of the top bin. I'm not sure how to implement this one.

File: factfinder/data/acs/2010/median.json [hunk 10]
```diff
@@ -702,6 +720,8 @@
     },
     "mdgr": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:723]: Again, I believe this should be true, but it looks like the top coding threshold is not the lower boundary of the top bin. I'm not sure how to implement this one.

File: factfinder/data/acs/2018/median.json [hunk 11]
```diff
@@ -1,6 +1,8 @@
 {
     "mdage": {
         "design_factor": 1.1,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:4]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 12]
```diff
@@ -98,6 +100,8 @@
     },
     "mdhhinc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:103]: Should be true.

File: factfinder/data/acs/2018/median.json [hunk 13]
```diff
@@ -167,6 +171,8 @@
     },
     "mdfaminc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:174]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 14]
```diff
@@ -236,6 +242,8 @@
     },
     "mdnfinc": {
         "design_factor": 1.5,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:245]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 15]
```diff
@@ -305,6 +313,8 @@
     },
     "mdewrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:316]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 16]
```diff
@@ -390,6 +400,8 @@
     },
     "mdemftwrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:403]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 17]
```diff
@@ -475,6 +487,8 @@
     },
     "mdefftwrk": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:490]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 18]
```diff
@@ -601,6 +617,8 @@
     },
     "mdvl": {
         "design_factor": 1.4,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:620]: Should be true

File: factfinder/data/acs/2018/median.json [hunk 19]
```diff
@@ -710,6 +728,8 @@
     },
     "mdgr": {
         "design_factor": 1.6,
+        "top_coding": false,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:731]: Should be true

**Review by mgraber — CHANGES_REQUESTED** [review]
I think the 2019 has same pattern as 2018

File: factfinder/calculate.py (hunk 1 above)
Comment (SPTKL) [reply factfinder/calculate.py]: oh right forgot about this

File: factfinder/data/acs/2010/median.json [hunk 20]
```diff
@@ -560,6 +574,8 @@
     },
     "mdrms": {
         "design_factor": 1.5,
+        "top_coding": true,
```
Comment (mgraber) [inline factfinder/data/acs/2010/median.json:577]: I think this is the only one not top coded.

File: factfinder/data/acs/2018/median.json [hunk 21]
```diff
@@ -560,6 +574,8 @@
     },
     "mdrms": {
         "design_factor": 1.5,
+        "top_coding": true,
```
Comment (mgraber) [inline factfinder/data/acs/2018/median.json:577]: This is false, I believe.

File: factfinder/data/acs/2019/median.json [hunk 22]
```diff
@@ -560,6 +574,8 @@
     },
     "mdrms": {
         "design_factor": 1.5,
+        "top_coding": true,
```
Comment (mgraber) [inline factfinder/data/acs/2019/median.json:577]: False

**Review by mgraber — APPROVED** [review]

**Pull request merged**